cleaners/geary.xml

```diff
@@ -0,0 +1,57 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!--
+
+    BleachBit
+    Copyright (C) 2008-2024 Andrew Ziem
+    https://www.bleachbit.org
+
+    This program is free software: you can redistribute it and/or modify
+    it under the terms of the GNU General Public License as published by
+    the Free Software Foundation, either version 3 of the License, or
+    (at your option) any later version.
+
+    This program is distributed in the hope that it will be useful,
+    but WITHOUT ANY WARRANTY; without even the implied warranty of
+    MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+    GNU General Public License for more details.
+
+    You should have received a copy of the GNU General Public License
+    along with this program.  If not, see <http://www.gnu.org/licenses/>.
+
+-->
+<cleaner id="geary" os="linux">
+  <label>Geary</label>
+  <description>Email client</description>
+  <running type="exe">geary</running>
+  <var name="accounts">
+    <value search="glob">~/.local/share/geary/*</value>
+    <value search="glob">$XDG_DATA_HOME/geary/*</value>
+    <value search="glob">~/.var/app/org.gnome.Geary/data/geary/*</value> <!-- Flatpak -->
+  </var>
+  <option id="cache">
+    <label>Cache</label>
+    <description>Delete the cache</description>
+    <action command="delete" search="walk.all" path="~/.cache/geary"/>
+    <action command="delete" search="walk.all" path="$XDG_CACHE_HOME/geary"/>
+    <action command="delete" search="walk.all" path="~/.var/app/org.gnome.Geary/cache"/> <!-- Flatpak -->
+  </option>
+  <option id="attachments">
+    <label>Attachments</label>
+    <description>Delete downloaded attachment files</description>
+    <!-- I use both the "walk.all" and "files" search options 
+         to print the files being removed and to remove the attachments directory. -->
+    <action command="delete" search="walk.all" path="$$accounts$$/attachments"/>
+    <action command="delete" search="file" path="$$accounts$$/attachments"/>
```

Comment:
@queeup 
walk.top: matches all contents of a directory and the top directory itself.

reference: https://docs.bleachbit.org/cml/cleanerml.html